data/adamas events.txt

```diff
@@ -39,9 +40,15 @@ event "at andol wormhole opens"
 			period 868.27
 			offset 191.403
 	system "Ed Andol"
-		object
-			sprite star/nova
-			period 10
+		object "Reminder Vortex"
```

Comment:
This fixes the disappearing station bug.

Reply:
I fixed the disappearing station bug with aba71ac443fac2f4f479f0cc4668b318f558b35c. The event failed to update the system including the station, effectively removing the station once the wormhole opened.

If I'm not mistaken all this does is removes the star and adds one side of the Reminder Vortex (dust wisp/dust wormhole) meant to be used later in the storyline (connecting to Scavenger space). Farther down this pull you remove the other side from Scavenger space, so it doesn't make sense to have one side of the wormhole here as an unlandable system object. I'm really sorry that I have a lot of legacy/deprecated system names associated with these two, but I scrapped a few different short questlines and system name schemes before the ones used now. I think after this pull goes through I'll likely redo and clarify the wormholes and wormhole events.

Reply:
I know you said you fixed it, but I downloaded the most recent version about a month ago and it still didn't work.

Reply:
You have to state all the system properties for i to not overwrite everything.